app/plugins/Overview/js/overview.js

```diff
@@ -3,25 +3,28 @@
 const IPC = require('ipc');
 // Library for arbitrary precision in numbers
 const BigNumber = require('bignumber.js');
+// Ensure precision
+BigNumber.config({ DECIMAL_PLACES: 24 });
+BigNumber.config({ EXPONENTIAL_AT: 1e+9 });
```

Comment:
From the BigNumber documentation:
>EXPONENTIAL_AT
number: integer, magnitude 0 to 1e+9 inclusive, or 
number[]: [ integer -1e+9 to 0 inclusive, integer 0 to 1e+9 inclusive ]
Default value: [-7, 20]
The exponent value(s) at which toString returns exponential notation.
```js
// Almost never return exponential notation:
BigNumber.config({ EXPONENTIAL_AT: 1e+9 })
```